src/lib/checkbox/checkbox.test.ts

```diff
@@ -0,0 +1,451 @@
+/// <reference types="mocha" />
```

Comment:
It would be worth removing this I think. It won't be needed in the future, and it will help with not accidentally leaving an `it.only` in here.